### DISCUSSION TITLE
Sorry for pushing to wrong branch

### DISCUSSION THREAD

**Discussion by th-we** [music-encoding/sibmei#226] [General]
@ahankinson @rettinghaus Sorry, I accidentally worked on the wrong branch and pushed to this repo. Sorry for pushing and force-pushing to undo that.

**Comment by rettinghaus** [comment]
I wouldn't have noticed. Which branch?

**Reply by th-we** [reply]
Interesting.  I accidentally pushed to the develop branch.
